packages/react-vapor/src/components/checkbox/examples/CheckboxExamples.tsx

```diff
@@ -1,65 +1,142 @@
+                </LabeledInput>
+            </Section>
+        </Section>
+    );
+};
+
+const GroupableCheckboxConnectedExamples: React.FunctionComponent = () => {
+    const toggleDisabledAllCheckboxes = (id: string, disabled?: boolean) => {
+        ReactVaporStore.dispatch(toggleDisabledAllGroupedCheckbox(id, undefined, disabled));
+    };
+    const parentId1 = 'parentId1';
+    const parentId2 = 'parentId2';
+
+    return (
+        <Section level={2} title="Groupable Checkboxes">
+            <Section>
+                <LabeledInput label="A groupable checkbox with some child checkboxes selected by default">
+                    <GroupableCheckboxConnected id={parentId1} isParent>
+                        <Label classes={['label']}>A parent checkbox connected</Label>
+                    </GroupableCheckboxConnected>
+                    <br />
+                    <GroupableCheckboxConnected id={parentId1 + 1} parentId={parentId1} defaultChecked={true}>
+                        <Label classes={['label', 'mr1']}>Checkbox child 1</Label>
+                    </GroupableCheckboxConnected>
+                    <GroupableCheckboxConnected id={parentId1 + 2} parentId={parentId1} defaultChecked={false}>
+                        <Label classes={['label', 'mr1']}>Checkbox child 2</Label>
+                    </GroupableCheckboxConnected>
+                    <GroupableCheckboxConnected id={parentId1 + 3} parentId={parentId1} defaultChecked={true}>
+                        <Label classes={['label', 'mr1']}>Checkbox child 3</Label>
+                    </GroupableCheckboxConnected>
+                    <GroupableCheckboxConnected id={parentId1 + 4} parentId={parentId1} defaultChecked={false}>
+                        <Label classes={['label']}>Checkbox child 4</Label>
+                    </GroupableCheckboxConnected>
+                </LabeledInput>
+                <br />
+            </Section>
+            <Section>
+                <LabeledInput label="A groupable checkbox with all checkboxes disabled or enabled">
+                    <div>
+                        <Button
+                            name={'Toggle checkboxes'}
+                            enabled={true}
+                            onClick={() => toggleDisabledAllCheckboxes(parentId2)}
+                        />
+                    </div>
+                    <GroupableCheckboxConnected id={parentId2} isParent defaultDisabled>
+                        <Label classes={['label']}>A parent checkbox connected</Label>
+                    </GroupableCheckboxConnected>
+                    <br />
+                    <GroupableCheckboxConnected
+                        id={parentId2 + 1}
+                        parentId={parentId2}
+                        defaultChecked={true}
+                        defaultDisabled
+                    >
+                        <Label classes={['label', 'mr1']}>Checkbox child 1</Label>
```

Comment:
There is kind of a pattern here.

* All checkbox labels have the `label` class
* All checkbox followed by another checkbox have `mr1`

Could it be put in CSS rules directly..?

Reply:
As it is It would more be a `mr4` and you are quite right about the pattern. I'll check that out.